### PR TITLE
Move azure page operation to a new folder

### PR DESCRIPTION
**Pull request by pshao25**
# Cadl Ranch Contribution Checklist:

- [ ] I have written a [scenario spec](../docs/writing-scenario-spec.md)
- [ ] I have **meaningful** `@scenario` names. Someone can look at the list of scenarios and understand what I'm covering.
- [ ] I have written a [mock API](../docs/writing-mock-apis.md)
- [ ] I have used `@scenarioDoc`s for extra scenario description and to tell people **how to pass** my mock api check.

### CODE REVIEW

File: packages/cadl-ranch-specs/http/azure/core/page/main.tsp [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+  ]
+}
+""")
+op listWithPage is global.Azure.Core.Foundations.Operation<{}, global.Azure.Core.Page<User>>;
+
+@doc("The parameters for listing users.")
+model ListItemInput {
+  @doc("The body of the input.")
+  @body
+  bodyInput: ListItemInputBody;
```
Comment (tadelesh) [inline packages/cadl-ranch-specs/http/azure/core/page/main.tsp:90]: Missed in previous review. I don't think we should put this aviod-rfc thing in the cadl-ranch test.

Comment (pshao25) [reply packages/cadl-ranch-specs/http/azure/core/page/main.tsp:90]: No you didn't. We discussed that. It is in ledger entry.

File: packages/cadl-ranch-specs/http/azure/core/page/main.tsp [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+{
+  "value":[
+     {
+        "id":1,
+        "name":"Madge",
+        "etag": "11bdc430-65e8-45ad-81d9-8ffa60d55b59"
+     }
+  ]
+}
+""")
```
Comment (timotheeguerin) [inline packages/cadl-ranch-specs/http/azure/core/page/main.tsp:83]:
````suggestion
```
""")
````

this is missing a closing brace

File: packages/cadl-ranch-specs/http/azure/core/page/main.tsp [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+{
+  "items":[
+     {
+        "id":1,
+        "name":"Madge",
+        "etag": "11bdc430-65e8-45ad-81d9-8ffa60d55b59"
+     }
+  ]
+}
+""")
```
Comment (timotheeguerin) [inline packages/cadl-ranch-specs/http/azure/core/page/main.tsp:168]: same here

**Review by timotheeguerin — APPROVED** [review]
This is just moving it without change right? There is just some code block that are missing closing ``` that were present before but I think would cause weird issue in the summary.md so be good to fix before merging

**Review by lmazuel — APPROVED** [review]

**Pull request merged**